subprojects/assemble/build-trace/src/main/kotlin/com/avito/android/build_trace/BuildTracePlugin.kt

```diff
@@ -29,11 +28,9 @@ public abstract class BuildTracePlugin : Plugin<Project> {
             project.layout.buildDirectory.dir("reports/build-trace")
         )
 
-        val loggerFactory = GradleLoggerPlugin.getLoggerFactory(project).get()
-
         val buildTraceListener = BuildTraceListener(
             output = outputDir.file("build.trace").get().asFile,
-            loggerFactory = loggerFactory
+            logger = project.logger
```

Comment:
When we really need logs we could use slf4j logger from Gradle or avito logger if that code is used from android runtime too